x/vm/wrappers/bank_test.go

```diff
@@ -150,24 +128,25 @@ func TestMintAmountToAccount(t *testing.T) {
 }
 
 func TestBurnAmountFromAccount(t *testing.T) {
+	eighteenDecimalsCoinInfo := testconstants.ExampleChainCoinInfo[testconstants.ExampleChainID]
+	sixDecimalsCoinInfo := testconstants.ExampleChainCoinInfo[testconstants.SixDecimalsChainID]
+
 	account := sdk.AccAddress([]byte("test_address"))
 
 	testCases := []struct {
-		name        string
-		evmDenom    string
-		evmDecimals uint8
-		amount      *big.Int
-		expectErr   string
-		mockSetup   func(*testutil.MockBankWrapper)
+		name      string
+		coinInfo  evmtypes.EvmCoinInfo
+		amount    *big.Int
+		expectErr string
+		mockSetup func(*testutil.MockBankWrapper)
 	}{
 		{
-			name:        "success - convert 18 decimals amount to 6 decimals",
-			evmDenom:    TokenDenom,
-			evmDecimals: 6,
-			amount:      big.NewInt(1e18),
-			expectErr:   "",
+			name:      "success - convert 18 decimals amount to 6 decimals",
```

Comment:
don't think we're converting anymore since we now have precisebank

Reply:
[ditto](https://github.com/cosmos/evm/pull/69#discussion_r2068352936)